lms/views/decorators/h_api.py

```diff
@@ -122,3 +124,124 @@ def wrapper(request, jwt):  # pylint: disable=too-many-branches
         return wrapped(request, jwt)
 
     return wrapper
+
+
+def create_h_group(wrapped):  # noqa: MC0002
+    """
+    Create a group in h if one doesn't exist already.
+
+    Call the h API to create a group for the LTI course, if one doesn't exist
+    already.
```

Comment:
Might be help to expand this docstring to indicate what it means for a group to "exist" (that is: "a record for it exists in the LTI app db"). Of course that doesn't mean it _actually_ exists, but you know what I mean...